README.md

```diff
@@ -55,3 +58,9 @@ npm version [<new-version> | major | minor | patch] -m "Release %s"
 
 ## License
 MIT
+
+
+[npm-image]: https://img.shields.io/npm/v/http-request-signature.svg?style=flat-square
```

Comment:
Why use the flat-square style? It doesn't match the Travis badge's style.

Reply:
I've done that because i've based myself on [itin-validator](https://github.com/uphold/ein-validator) repository

Reply:
Sure, but there both badges use the flat square style, so it's consistent.